src/adapter/auth-oso-rebac/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "kamu-adapter-auth-oso-rebac"
+description = "Authorization adapter for Kamu domain based on OSO library & ReBAC properties"
+version = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license-file = { workspace = true }
+keywords = { workspace = true }
+include = { workspace = true }
+edition = { workspace = true }
+publish = { workspace = true }
+
+
+[lints]
+workspace = true
+
+
+[lib]
+doctest = false
+
+[dependencies]
+database-common = { workspace = true }
+init-on-startup = { workspace = true }
+internal-error = { workspace = true }
+kamu-accounts = { workspace = true }
+kamu-auth-rebac = { workspace = true }
+kamu-auth-rebac-services = { workspace = true }
```

Comment:
Needed both in main and dev dependencies?

Reply:
Eagle eye